learning/tour-of-beam/frontend/lib/pages/welcome/screen.dart

```diff
@@ -212,6 +200,34 @@ class _SdkButtons extends StatelessWidget {
   static const String _sdk = 'Java';
 }
 
+class _SdkButton extends StatelessWidget {
+  final String sdk;
+  final String selectedSdk;
```

Comment:
If we mimic the Flutter `Radio`, the properties should be `value`, `groupValue`, and `onChanged`: https://api.flutter.dev/flutter/material/Radio-class.html

`_SdkButtons` should also have `onChanged` property so as to not handle the change by itself.